spec/system/admin/payments_stripe_spec.rb

```diff
@@ -18,9 +18,11 @@
     create(:stripe_account, enterprise: order.distributor, stripe_user_id: "abc123")
   end
 
-  before { setup_stripe }
+  around do |example|
+    with_stripe_setup { example.run }
+  end
 
-  context "making a new Stripe payment", js: true do
+  context "making a new Stripe payment" do
```

Comment:
I'm not sure, but it looks like we have some Stripe-related Javascript about selecting a card, so it might be worth keeping JS mode to ensure that is covered.

Reply:
@jibees Do you have any thoughts to add to this?

Reply:
I re-read the documentation of capybara (https://github.com/teamcapybara/capybara): 

> Use js: true to switch to the Capybara.javascript_driver (:selenium by default), or provide a :driver option to switch to one specific driver.

But we use https://github.com/rubycdp/cuprite as our javascript driver: 

https://github.com/openfoodfoundation/openfoodnetwork/blob/911d816233bb5426610e497562ec6ab3c969e82a/spec/system/support/cuprite_setup.rb#L25


So my opinion: we should remove this `js: true` on all system spec since it's useless.

Reply:
Yes, that's my understanding too - we should remove it from all specs, it's a tech debt for system specs.

Reply:
Oh, thanks for pointing this out! So the default is always use a browser test (Chrome via Cuprite) for _all_ system tests. Hence it's unnecessary to specify JS mode.